spec/acceptance/suites/default/00_default_spec.rb

```diff
@@ -65,39 +72,33 @@
       end
 
       it 'can login to 389DS' do
-        on(host, %(ldapsearch -x -y "#{root_dn_password_file}" -D "cn=Directory_Manager" -h localhost -b "cn=tasks,cn=config"))
+        on(host, %(ldapsearch -x -w "#{rootpasswd}" -D "#{root_dn}" -h localhost -b "cn=tasks,cn=config"))
       end
 
       it 'can login to 389DS via LDAPI' do
-        on(host, %(ldapsearch -x -y "#{root_dn_password_file}" -D "cn=Directory_Manager" -H ldapi://%2fvar%2frun%2fslapd-#{ds_root_name}.socket -b "cn=tasks,cn=config"))
+        on(host, %(ldapsearch -x -w "#{rootpasswd}" -D "#{root_dn}" -H ldapi://%2fvar%2frun%2fslapd-#{ds_root_name}.socket -b "cn=tasks,cn=config"))
       end
 
-      it 'contains the default entries' do
-        domain = on(host, %(puppet apply --color=none -e '$dn = simplib::ldap::domain_to_dn($facts["domain"], true); notice("DOMAIN => ${dn}")'))
-                 .stdout
-                 .lines
-                 .grep(%r{DOMAIN =>})
-                 .first
-                 .split('=> ')
-                 .last
-                 .strip
+      it 'contains the entries from the ldif' do
 
-        result = on(host, %(ldapsearch -x -y "#{root_dn_password_file}" -D "cn=Directory_Manager" -H ldapi://%2fvar%2frun%2fslapd-#{ds_root_name}.socket -b "#{domain}")).output
+        result = on(host, %(ldapsearch -x -w "#{rootpasswd}" -D "#{root_dn}" -H ldapi://%2fvar%2frun%2fslapd-#{ds_root_name}.socket -b "#{base_dn}")).output
 
-        expect(result.lines.grep(%r{cn=administrators,ou=Group,#{domain}})).not_to be_empty
+        expect(result.lines.grep(%r{cn=administrators,ou=Group,#{base_dn}})).not_to be_empty
       end
 
       it 'fails when logging in with forced encryption' do
-        expect { on(host, %(ldapsearch -ZZ -x -y "#{root_dn_password_file}" -D "cn=Directory_Manager" -h `hostname -f` -b "cn=tasks,cn=config")) }.to raise_error(Beaker::Host::CommandFailure)
+        expect { on(host, %(ldapsearch -ZZ -x -w "#{rootpasswd}" -D "#{root_dn}" -h `hostname -f` -b "cn=tasks,cn=config")) }.to raise_error(Beaker::Host::CommandFailure)
       end
     end
 
     context 'with an instance to delete' do
+      let(:root_dn_password_file) do
```

Comment:
In other parts of the test, you use the password directly, in lieu of the password file.  Why did you leave use of the password file here?

Reply:
In that instance it autogenerates the password using passgen and I don't have the password handy.  I can change it to not auto-generate it.  Do you think that is necessary?